geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -1737,7 +1737,6 @@ private void extractDeltaIntoEvent(Object value, EntryEventImpl event) {
               throw new DeltaSerializationException("Caught exception while sending delta", e);
             }
             event.setDeltaBytes(hdos.toByteArray());
-            getCachePerfStats().endDeltaPrepared(start);
```

Comment:
With the removal of this line, the `start` variable on line 1731 can now be removed as it's no longer used.